optuna/visualization/matplotlib/_param_importances.py

```diff
@@ -27,26 +21,10 @@
     from optuna.visualization.matplotlib._matplotlib_imports import Axes
     from optuna.visualization.matplotlib._matplotlib_imports import cm
     from optuna.visualization.matplotlib._matplotlib_imports import plt
-    from optuna.visualization.matplotlib._matplotlib_imports import Rectangle
 
-    # Color definitions.
+    # fix single color for bars
     cmap = cm.get_cmap("tab20c")
-    _distribution_colors = {
-        UniformDistribution: cmap(0),
-        LogUniformDistribution: cmap(0),
-        DiscreteUniformDistribution: cmap(0),
-        IntUniformDistribution: cmap(1),
-        IntLogUniformDistribution: cmap(1),
-        CategoricalDistribution: cmap(3),
-    }
-    _legend_elements = [
-        Rectangle([0, 0], 0, 0, facecolor=cmap(0), label="Uniform Distribution"),
-        Rectangle([0, 0], 0, 0, facecolor=cmap(0), label="Log Uniform Distribution"),
-        Rectangle([0, 0], 0, 0, facecolor=cmap(0), label="Discrete Uniform Distribution"),
-        Rectangle([0, 0], 0, 0, facecolor=cmap(1), label="Int Uniform Distribution"),
-        Rectangle([0, 0], 0, 0, facecolor=cmap(1), label="Int Log Uniform Distribution"),
-        Rectangle([0, 0], 0, 0, facecolor=cmap(3), label="Categorical Distribution"),
-    ]
+    bar_color = cmap(0)
```

Comment:
Same as https://github.com/optuna/optuna/pull/2576/files#r613301532.
I think we can pass `cmap(0)` directory to `color` in `ax.barh`.